cfme/tests/containers/test_pause_resume_workers.py

```diff
@@ -94,6 +104,10 @@ def test_pause_and_resume_single_provider_api(appliance, provider, from_collecti
     assert evm_tail_disable.validate()
     # Verify all monitoring workers have been shut down
     assert wait_for(lambda: not check_ems_state_in_diagnostics(appliance, provider))
+    # Pausing is an asynchronous call. It takes a bit for the provider to full pause. There is no
+    # log that indicates that the provider is full paused so sleeping 15 seconds for now.
+    # Enhancement BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1752642
```

Comment:
As, we try to avoid `time.sleep` in our tests/ framework. I think better to add `TODO(BZ-1752642): <todo message>` to remove this `time.sleep` so it will be easy to search.